goose3/crawler.py

```diff
@@ -272,6 +272,7 @@ def get_html(self, crawl_candidate, parsing_candidate):
             encodings = get_encodings_from_content(response.text)
             if len(encodings) > 0:
                 self.article._meta_encoding = encodings[0]
+                response.encoding = encodings[0]
```

Comment:
I could be wrong, but wouldn't [this](https://github.com/goose3/goose3/pull/92/files#diff-b467c6f23c4c38313d233de3f2db12eaR266) be the path taken?

Reply:
So, if requests gives it the 'ISO-8859-1' encoding, we want to check again, otherwise we want to see if there is encoding information in the page that we can use to help. I am not sure which path you are referring to.